src/utils/stark/starkCurve.ts

```diff
@@ -100,3 +109,58 @@ export function generateStarkPrivateKey(): string {
   const keyPair = starkEc.genKeyPair();
   return grindKey(keyPair.getPrivate(), starkEcOrder);
 }
+
+function getIntFromBits(
+  hex: string,
+  start: number,
+  end: number | undefined = undefined,
+): number {
+  const bin = encUtils.hexToBinary(hex);
+  const bits = bin.slice(start, end);
+  const int = encUtils.binaryToNumber(bits);
+  return int;
+}
+
+function getAccountPath(
```

Comment:
This method is copied over from v0.7 and earlier. I think we should keep the existing method name here so that it is easier to tract through version history.